.circleci/config.yml

```diff
@@ -97,6 +151,8 @@ workflows:
           app_src_directory: client/java
       - test-client-python
       - test-integration-common
+      - build-integration-spark
+      - integration-test-spark
```

Comment:
The `integration-test-spark` CI job should depend on `build-integration-spark`, see https://github.com/MarquezProject/marquez/blob/main/.circleci/config.yml#L339

Reply:
You mean that we don't need to run Integration tests if build failed? Cause it's not dependent directly, we can run ITs without build, task run shadowJar and then run tests. I'll change it in the same way as it was